### ISSUE TITLE
mysql version doesn't match dreamhost's

### ISSUE THREAD

**Issue by acodispo** [goodguyry/dreambox#36]
Apparently the version of MySQL being used on DH shared servers is `5.6.34`. Dreambox has `5.5.40`.

On the dreambox:

```
mysql -u root -e "SELECT @@version";
+------------+
| @@version  |
+------------+
| 5.5.40-log |
+------------+
```

On Dreamhost server:

```
mysql -u myuser -p -h mysql.myserver.com -e "SELECT @@version;"
Enter password: 
+------------+
| @@version  |
+------------+
| 5.6.34-log |
+------------+
```

**Comment by consumr** [comment]
When I run mysql --version, it returns 
"mysql  Ver 14.14 Distrib 5.5.58, for debian-linux-gnu (x86_64) using readline 6.3"
When I run "select version();" from phpmyadmin, it returns: 5.6.34-log. 

...just an FYI, really, because it looks like the best version to use is 5.6, but there appears to be a mismatch between the shared hosting ssh user's version and the servers that are actually hosting mysql.

**Comment by goodguyry** [comment]
This is fixed in v0.3.0
